core/services/feeds/service.go

```diff
@@ -1607,14 +1607,15 @@ func (s *service) isApprovable(ctx context.Context, propStatus JobProposalStatus
 	case SpecStatusRevoked:
 		return errors.New("cannot approve a revoked spec")
 	case SpecStatusCancelled:
-		// Allowed to approve a cancelled job if it is the latest job
-		latest, serr := s.orm.GetLatestSpec(ctx, proposalID)
+		// allow approval of a cancelled job if no other spec is approved
```

Comment:
So if an job has an approved version (i.e. it is already running) this means that trying to approve another job version will fail. 

Just want to make sure I understand that correctly